### ISSUE TITLE
Bicycle route ignored oneway tag

### ISSUE THREAD

**Issue by tomhughes** [tomhughes/trac-tickets#3023]
**[Submitted to the original trac issue database at 10.44am, Monday, 31st May 2010]**

Used http://www.yournavigation.org/ to explore local route. When I selected bicycle, the route
ignored a one way tag. (Tag was respected for cars, etc). The way data is 

= Tags: = 	
hgv = no[[BR]]
highway = unclassified[[BR]]
name = Narrow Hill[[BR]]
note = hgv=no is not official, just practical! [[BR]]
oneway = yes[[BR]]
== Nodes: == 	
10204767 (also part of ways Woodgreen (3032635) and Woodgreen (3032634))
158641245
158641255 (also part of way 15808343)
158641262
158641265
158641269 (also part of way 17505773)
181316634 (also part of way Woodgreen (17505772))
158641273
5660817
158641277
5660816 (also part of way West End (3015021))

**Comment by tomhughes** [comment]
**Author: nroets@gmail.com**
**[Added to the original trac issue at 12.32pm, Monday, 31st May 2010]**

It could be that the router wants the cyclist to dismount and walk up the street. Gosmore and YN does not present that data to the user, so you will have to speculate if this was intended.

But I think there still is a bug.

**Comment by tomhughes** [comment]
**Author: messpert**
**[Added to the original trac issue at 3.27pm, Monday, 31st May 2010]**

I hadn't thought of that :-)

**However**, I have just tried this on gosmore running on a satnav (Windoze CE) and
it worked! So perhaps the gosmore that I have just installed (from the prebuilt gosmoreWinMob.zip)
has fixed the bug. I think that perhaps I should close the bug given that result. I thought that YN used gosmore?
